src/struct_pack/examples/serialize/go_deserialize.go

```diff
@@ -0,0 +1,80 @@
+package main
```

Comment:
add ci for this demo is better

Reply:
ok,I will change package name and add test for this demo